tests/e2e/script_test.go

```diff
@@ -228,18 +231,21 @@ func TestE2EScript(t *testing.T) {
 				}
 
 				if os.Getenv("GOLDEN_REQUEST_CHECKS") != "" || os.Getenv("WRITE_GOLDEN_OUTPUT") != "" {
-					x := NewNormalizer(uniqueID, project)
+					if os.Getenv("KCC_USE_DIRECT_RECONCILERS") != "" && os.Getenv("GOLDEN_REQUEST_CHECKS") != "" {
+						h.Logf("ignoring http log checking for now for direct resources")
```

Comment:
I do have a plan to check the golden logs but the change may be a bit more involved.

AFAICT, when we check two golden logs:

- `non_direct_http.log` [1]
- `direct_http.log` [2]

The following needs to be checked at least:
- we want to check that a POST/ PUT  in [2] exists in [1] AND ignore any GETs in [1] before it.
- we want to check that a GET in [2] has at least one corresponding GET in [1].

This excludes some further normalization like ignoring user agents.

IMO I think we get a lot of validation from checking the KRM objects themselves already.